ws.py

```diff
@@ -116,15 +116,14 @@ def pathwayChoices():
     }
 
 
-def pathwaySelection():
-    choices = pathwayChoices()
+def globalPathwayChoices():
+    # TODO fix slider settings
```

Comment:
should this be fixed here or in follow up?

Reply:
done in 5ca686fb416c2509f377a111ef5ab39d360e2135